Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -8,7 +9,85 @@ namespace Poker.BE.Domain.Game
 {
     public class Round
     {
-        // TODO: complete - set team member to do this
 
+        #region Enums
+        public enum Move
+        {
+            check,
+            call,
+            bet,
+            fold,
+            raise,
+            allin
+        }
+        #endregion
+
+        #region fields
+        private ICollection<Player> activeUnfoldedPlayers;
+        private Turn currentTurn;
+        private Player dealer;
+        private Player currentPlayer;
+        #endregion
+
+        #region Constructors
+        public Round(Player dealer, ICollection<Player> activeUnfoldedPlayers)
+        {
+            this.dealer = dealer;
+            this.currentPlayer = this.activeUnfoldedPlayers.ElementAt((activeUnfoldedPlayers.ToList().IndexOf(dealer)+2)%activeUnfoldedPlayers.Count);
+            this.activeUnfoldedPlayers = activeUnfoldedPlayers;
+            this.currentTurn = new Turn(currentPlayer);
+        }
+        #endregion
+
+        #region Methods
+        public void PlayMove(Move playMove)
+        {
+            switch (playMove)
+            {
+                case Move.check :
+                    {
```

Comment:
yes, but my role of thumb in  curly barracks is: "If its looks good, use them".
this may fix a lot of "if (condition) do" things that may be grows in the future to "if (condition) do do more" and forgot the {}